scripts/build/TestFx.Loc.targets

```diff
@@ -9,6 +9,7 @@
   <!-- Default Localization folder. -->
   <PropertyGroup>
     <ResourceDirectory>$(ProjectDir)Resources</ResourceDirectory>
+    <SharedResourceDirectory>$(ProjectDir)..\PlatformServices.Shared\netstandard1.3\Resources</SharedResourceDirectory>
```

Comment:
Remove all the loc related stuff, as per the staging plan

Reply:
As discussed, keeping shared resources for now as changes are needed only at 1 place.